client/src/components/StyledText.tsx

```diff
@@ -155,6 +155,14 @@ function computeBasicKeywordData() {
                 replacement: data.replacement === undefined ? undefined : translate(data.replacement)
             }
         });
+        for (let i = 0, variant; (variant = translateChecked(`${keyword}:var.${i}`)) !== null; i++) {
+            KEYWORD_DATA_MAP[variant] = (Array.isArray(data) ? data : [data]).map(data => {
+                return {
+                    ...data,
+                    replacement: data.replacement === undefined ? undefined : translate(data.replacement)
+                }
+            });
+        }
```

Comment:
You should use the entire lang file instead of just keywrods.
Having to make a different entry into keywords.json for every word would be ridiculous
My attempt to make this work for visits didnt work.
It should also work for roleblocked and alot of other things